File: examples/dummy-java/lifecycle/AnnounceServer.java
public class AnnounceServer {
  public static String lifecycleEvent = 'onConnect';
  public static String[] servers = new String[]{'mosquitto'};
  public EventResponse onEvent(EventRequest event){
    EventResponse response = new EventResponse();
    Send s = new Send();
    s.server = 'mosquitto';
    s.channel = 'server/announce';
    Paylodssss p = new Paylodssss();
    p.id = 'test'; // Date.now()
    s.payload = p;
    response.send = new SendEvent[]{
      s
    }
  }
}

File: packages/glee/examples/dummy/lifecycle/announceServer.ts
export default async function () {
  return {
    send: [{
      server: 'mosquitto',
      channel: 'serverAnnounce',
      payload: {
        id: process.env.SERVER_ID || String(Date.now()),
      }
    }]
  }
}

export const lifecycleEvent = 'onConnect'
export const servers = ['mosquitto']
